cmd/portable-package.rb

```diff
@@ -37,8 +37,8 @@ def run
         args.named.each do |name|
           name = "portable-#{name}" unless name.start_with? "portable-"
           begin
-            # On Linux, install glibc@2.13 and linux-headers from bottles and don't install their build dependencies.
-            bottled_dep_allowlist = %w[glibc@2.13 linux-headers@4.4]
+            # On Linux, install glibc and linux-headers from bottles and don't install their build dependencies.
+            bottled_dep_allowlist = %w[glibc@2.13 glibc@2.17 linux-headers@4.4]
```

Comment:
Might be nicer to just check for `glibc@*` or `linux-headers@*` rather than hardcoding here